koocook_core/tests/utils.py

```diff
@@ -0,0 +1,122 @@
+from typing import Iterable, List
+import random
+from decimal import Decimal
+
+import numpy as np
```

Comment:
Do we really need `numpy` as another dependency?

Reply:
Could make an effort to remove them. Just there for some performance optimization. I'm not if the function is used anymore though, lemme look at it first

Reply:
Then I'll leave this reply as is to remind you.

Reply:
Resolved in 5b6bd54afaa842a3e074eac001de110f4bff00d2